migrations/v7.js

```diff
@@ -0,0 +1,161 @@
+
+  whereContent('adapt-contrib-accordion - where accordion', async content => {
+    accordions = getComponents('accordion');
+    return accordions.length;
+  });
+
+  mutateContent('adapt-contrib-accordion  - update default mobileInstruction text', async () => {
+    accordions.forEach(accordion => {
+      if (!_.has(accordion, 'mobileInstruction')) _.set(accordion, 'mobileInstruction', originalMobileInstruction);
+      if (accordion.mobileInstruction === originalMobileInstruction || '') {
+        accordion.mobileInstruction = 'Select the headings to find out more.';
+      }
+    });
+    return true;
+  });
+
+  checkContent('adapt-contrib-accordion - check mobileInstruction attribute', async () => {
+    const isInvalid = accordions.find(({ mobileInstruction }) => mobileInstruction === originalMobileInstruction);
+    if (isInvalid) throw new Error('adapt-contrib-accordion - mobileInstruction attribute not modified');
+    return true;
+  });
+
+  updatePlugin('adapt-contrib-accordion - update to v7.4.0', { name: 'adapt-contrib-accordion', version: '7.4.0', framework: '>5.20.1' });
+
+  testSuccessWhere('correct version with accordion components with default/custom/no mobileInstruction', {
+    fromPlugins: [{ name: 'adapt-contrib-accordion', version: '7.3.0' }],
+    content: [
+      { _id: 'c-100', _component: 'accordion', mobileInstruction: originalMobileInstruction },
+      { _id: 'c-105', _component: 'accordion', mobileInstruction: 'Custom mobileInstruction' },
+      { _id: 'c-110', _component: 'accordion' }
+    ]
+  });
+
+  testStopWhere('no accordion components', {
+    fromPlugins: [{ name: 'adapt-contrib-accordion', version: '7.3.0' }],
+    content: [{ _component: 'other' }]
+  });
+
+  testStopWhere('incorrect version', {
+    fromPlugins: [{ name: 'adapt-contrib-accordion', version: '7.4.0' }]
+  });
+});
+
+describe('adapt-contrib-accordion - v7.4.0 > v7.7.0', async () => {
+  let accordions;
+
+  whereFromPlugin('adapt-contrib-accordion - from v7.4.0', { name: 'adapt-contrib-accordion', version: '<7.7.0' });
+
+  whereContent('adapt-contrib-accordion - where accordion', async content => {
+    accordions = getComponents('accordion');
+    return accordions.length;
+  });
+
+  mutateContent('adapt-contrib-accordion - add accordion._items._titleIcon', async () => {
+    accordions.forEach(accordion => {
+      accordion._items.forEach(item => {
+        item._titleIcon = '';
+      });
+    });
+    return true;
+  });
+
+  mutateContent('adapt-contrib-accordion - add accordion._isCenterAligned', async () => {
+    accordions.forEach(accordion => {
+      accordion._isCenterAligned = false;
+    });
+    return true;
+  });
+
+  checkContent('adapt-contrib-accordion - check accordion._items._titleIcon attribute', async () => {
+    const isValid = accordions.every(accordion =>
+      accordion._items.every(item =>
+        item && item._titleIcon !== undefined
+      )
```

Comment:
Optional chaining or `_.has`
```suggestion
      accordion._items.every(item => _.has(item, '_titleIcon')
```

Reply:
Used optional chaining to stay consistent with previous comment.